Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
+
+    if (var/"postgres").exist?
+      mv var/"postgres", var/"postgres-#{version_suffix}"
```

Comment:
Well it requires manual intervention to upgrade the database which we're not doing here anyway?